src/code/components/notifications.js

```diff
@@ -21,10 +21,10 @@ class Notifications extends React.Component {
 
   handleClose(onCloseButton, onCloseNotifications) {
     return () => {
+      onCloseNotifications();
```

Comment:
Now closing notifications before any close button actions, so that narration created by the action isn't immediately cleared.